documentation/docs/index.md

```diff
@@ -81,6 +81,33 @@ Files[IO]
   .drain
 ```
 
+### Dealing with erroneous files
+
+The default behaviour when parsing CSV files, is to terminate the stream whenever the columns of a row do not match 
+with the columns of the header. If you're dealing with CSV files that could contain these kind of errors, you can make 
+use of the `lenient` package. You will get back a `List` of results, where each parsed row is represented by an 
```

Comment:
```suggestion
use of the `lenient` package. You will get back a `Stream` of results, where each parsed row is represented by an 
```

It's only a `List` because you use `toList` in the example, most real use cases will actually transform the results within a stream to stay in constant memory world.